tests/test_integration.py

```diff
@@ -66,15 +89,16 @@ def test_train_and_run(
         str(output_filename),
         str(mgf_small),
         str(mzml_small),
+        "--evaluate",
     ]
 
-    result = run(predict_args)
+    result = run(eval_args)
     assert result.exit_code == 0
     assert output_filename.is_file()
 
     mztab = pyteomics.mztab.MzTab(str(output_filename))
     # Verify that both input peak files are listed in the metadata.
-    for i, filename in enumerate(["small.mgf", "small.mzml"], 1):
+    for i, filename in enumerate(["small.mgf"], 1):
```

Comment:
**nitpick:** The for loop could be removed, because only a single file needs to be checked.